pkg/client/leaderelection/leaderelection_test.go

```diff
@@ -55,18 +55,18 @@ func TestTryAcquireOrRenew(t *testing.T) {
 		{
 			reactors: []struct {
 				verb     string
-				reaction testcore.ReactionFunc
+				reaction core.ReactionFunc
 			}{
 				{
 					verb: "get",
-					reaction: func(action testcore.Action) (handled bool, ret runtime.Object, err error) {
-						return true, nil, errors.NewNotFound(action.(testcore.GetAction).GetResource().GroupResource(), action.(testcore.GetAction).GetName())
+					reaction: func(action core.Action) (handled bool, ret runtime.Object, err error) {
+						return true, nil, errors.NewNotFound(api.Resource(action.(core.GetAction).GetResource().Resource), action.(core.GetAction).GetName())
```

Comment:
Old code was `action.(testcore.GetAction).GetResource().GroupResource()` which pulls both group and resource from the action.  

New code has `api.Resource(action.(core.GetAction).GetResource().Resource` which keeps the resource part, but stomps the group part to `api.GroupName`.